lib/redux/actions/async.js

```diff
@@ -155,6 +155,13 @@ export function doLogin(creds, opts) {
         profile: results[1],
         memberships: results[2]
       }));
+
+      // detect if a VCA here and redirect to clinic user select screen
+
+      if(actionUtils.userHasRole(results[0].user, 'clinic')){
```

Comment:
Perhaps we should unify the `userHasRole` function under the new `personUtils`? I think it makes more sense to keep everything relating to info we're extracting from a user object in one place.